HealthcareSystem/PatientWebApp/Controllers/ExaminationController.cs

```diff
@@ -165,5 +165,22 @@ public ActionResult GetFollowingExaminationsByPatient(string patientJmbg)
                 return StatusCode(500, e.Message);
             }
         }
+
+
+        [HttpGet("examination-by-id/{examinationId}")]
+        public ActionResult GetExaminationById(int examinationId)
```

Comment:
This isn't in accordance with [RESTful resource naming](https://restfulapi.net/resource-naming/), it should be just "examinations/{id}" and not "examinations/examination-by-id/{id}".